scss/_settings_font.scss

```diff
@@ -11,3 +11,4 @@ $font-sizes: (
   base: 1rem,
   big: $font-size-largescreen
 ) !default;
+$font-weight-bold: 400;
```

Comment:
Shouldn't this have  !default at the end?

Reply:
```suggestion
$font-weight-bold: 400 !default;
```